lib/node_modules/@stdlib/blas/base/drot/docs/repl.txt

```diff
@@ -0,0 +1,134 @@
+
+    Returns
+    -------
+    y: Float64Array
+        Second input array.
+
+    Examples
+    --------
+    // Standard Usage:
+    > var x = new {{alias:@stdlib/array/float64}}( [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+    > var y = new {{alias:@stdlib/array/float64}}( [ 6.0, 7.0, 8.0, 9.0, 10.0 ] );
+    > {{alias}}( x.length, x, 1, y, 1, 0.8, 0.6 );
+    > x
+    <Float64Array>[ ~4.4, ~5.8, 7.2, 8.6, 10.0 ]
+    > y
+    <Float64Array>[ ~4.2, 4.4, 4.6, 4.8, 5.0 ]
+
+    // Advanced Indexing:
+    > x = new {{alias:@stdlib/array/float64}}( [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+    > y = new {{alias:@stdlib/array/float64}}( [ 6.0, 7.0, 8.0, 9.0, 10.0 ] );
+    > {{alias}}( 2, x, -1, y, 2, 0.707, 0.707 );
+    > x
+    <Float64Array>[ ~6.36, ~5.66, 3.0, 4.0, 5.0 ]
+    > y
+    <Float64Array>[ ~2.83, 7.0, ~4.95, 9.0, 10.0 ]
+
+    // Using typed array views:
+    > var x0 = new {{alias:@stdlib/array/float64}}( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+    > var y0 = new {{alias:@stdlib/array/float64}}( [ 7.0, 8.0, 9.0, 10.0, 11.0, 12.0 ] );
+    > var x1 = new {{alias:@stdlib/array/float64}}( x0.buffer, x0.BYTES_PER_ELEMENT*1 );
+    > var y1 = new {{alias:@stdlib/array/float64}}( y0.buffer, y0.BYTES_PER_ELEMENT*3 );
+    > {{alias}}( 3, x1, 1, y1, 1, 0.8, 0.6 );
+    > x0
+    <Float64Array>[ 1.0, 7.6, 9.0, ~10.4, 5.0, 6.0 ]
+    > y0
+    <Float64Array>[ 7.0, 8.0, 9.0, 6.8, ~7.0, ~7.2 ]
+
+
+{{alias}}.ndarray( N, x, strideX, offsetX, y, strideY, offsetY, c, s )
+    Applies a plane rotation using alternative indexing semantics.
+
+    While typed array views mandate a view offset based on the underlying
+    buffer, the offset parameters support indexing semantics based on starting
+    indices.
+
+    Parameters
+    ----------
+    N: integer
+        Number of indexed elements.
+
+    x: Float64Array
+        First input array.
+
+    strideX: integer
+        Index increment for `x`.
+
+    offsetX: integer
+        Starting index for `x`.
+
+    y: Float64Array
+        Second input array.
+
+    strideY: integer
+        Index increment for `y`.
+
+    offsetY: integer
+        Starting index for `y`.
+
+    c: number
+        Cosine of the angle of rotation.
+
+    s: number
+        Sine of the angle of rotation.
+
+    Returns
+    -------
+    y: Float64Array
+        Second input array.
+
+    Examples
+    --------
+    // Standard Usage:
+    > var x = new {{alias:@stdlib/array/float64}}( [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+    > var y = new {{alias:@stdlib/array/float64}}( [ 6.0, 7.0, 8.0, 9.0, 10.0 ] );
+    > {{alias}}.ndarray( x.length, x, 1, 1, y, 1, 1, 0.8, 0.6 );
+    > x
+    <Float64Array>[ 1.0, ~5.8, 7.2, 8.6, 10.0 ]
+    <Float64Array>[ 6.0, 4.4, ~4.6, ~4.8, 5.0 ]
+
+    // Advanced Indexing:
+    > x = new {{alias:@stdlib/array/float64}}( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+    > y = new {{alias:@stdlib/array/float64}}( [ 7.0, 8.0, 9.0, 10.0, 11.0, 12.0 ] );
+    > {{alias}}.ndarray( x.length, x, 2, 2, y, 2, 2, 0.8, 0.6 );
```

Comment:
```suggestion
    > {{alias}}.ndarray( 3, x, 2, 1, y, 2, 1, 0.8, 0.6 );
```

Same comment. I encourage you to think a bit more carefully about what the various parameters mean within the context of array iteration. The errors in these examples have common themes.